packages/nx/src/native/tests/command.spec.ts

```diff
@@ -26,23 +27,26 @@ describe('runCommand', () => {
   it('should subscribe to output', (done) => {
     const childProcess = runCommand('echo "hello world"', process.cwd());
 
-    childProcess.onOutput((output) => {
-      expect(output.trim()).toEqual('hello world');
+    let output = '';
+    childProcess.onOutput((chunk) => {
+      output += chunk;
     });
-
+    
```

Comment:
this might fail prettier?

Reply:
🤔 it pushed up without no verify, but I'll fix it while I remove the import.

Reply:
Ah, there was a prettierignore rule hitting these spec files. I adjusted it.